modules/polygon/test/earcut-testdata/expected.js

```diff
@@ -0,0 +1,57 @@
+export default {
```

Comment:
We usually use the following directory structure for test data `modules/polygon/test/data/earcut` (in particular see loaders.gl)

That also allows us to have a README.md in that directory with attribution of the files.

Reply:
Has been addressed in https://github.com/uber-web/math.gl/pull/234